<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NSS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="world_info_frame">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>630</y>
      <width>220</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="pop_val">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="This world's age"/>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pop_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="How many organisms are in this world"/>
     </property>
     <property name="text">
      <string>Population:</string>
     </property>
    </widget>
    <widget class="QLabel" name="co2_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="The amount of CO2 in this world"/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>CO2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="co2_val">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_val">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="world_controls_frame">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>271</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="add_co2_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Adds CO2 into the world"/>
     </property>
     <property name="text">
      <string>Add CO2</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="co2_spinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Amount of CO2 to add"/>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="rem_co2_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Removes CO2 from the world"/>
     </property>
     <property name="text">
      <string>Remove CO2</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rem_co2_spinbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Amount of CO2 to remove"/>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="set_co2_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Sets the world's CO2 to a specific value"/>
     </property>
     <property name="text">
      <string>Set CO2</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="New CO2 value"/>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1800</number>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="organismView">
    <property name="geometry">
     <rect>
      <x>556</x>
      <y>630</y>
      <width>141</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="organism_controls_frame">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>630</y>
      <width>165</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="kill_btn">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>55</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Kill this organism"/>
     </property>
     <property name="text">
      <string>Kill</string>
     </property>
    </widget>
    <widget class="QPushButton" name="feed_btn">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>40</y>
       <width>55</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Feed this organism"/>
     </property>
     <property name="text">
      <string>Feed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_brain_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>75</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="View this organism's brain"/>
     </property>
     <property name="text">
      <string>View Brain</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_organism_btn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Save this organism to a file"/>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hurt_btn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Take energy from this organism"/>
     </property>
     <property name="text">
      <string>Hurt</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="organism_info_frame">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>630</y>
      <width>181</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="energy_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="How much energy this organism has"/>
     </property>
     <property name="text">
      <string>Energy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="generation_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="This organism's generation"/>
     </property>
     <property name="text">
      <string>Generation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="neurons_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="How many neurons are in this organism's brain"/>
     </property>
     <property name="text">
      <string>Neurons:</string>
     </property>
    </widget>
    <widget class="QLabel" name="energy_val">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="generation_val">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="neurons_val">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="worldView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1071</width>
      <height>609</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
